modules/c/nitf/source/BandSource.c

```diff
@@ -55,8 +55,9 @@ NITFPRIV(NITF_BOOL) MemorySource_contigRead(
         MemorySourceImpl* memorySource,
         void* buf,
         nitf_Off size,
-        nitf_Error* error)
+        nitf_Error* e)
 {
+	(void) e; //Suppresses a warning. Param seems to exist for consistency purposes. Not used.
```

Comment:
Is there a reason to rename to `e` rather than `error`?  Try to avoid one character variable names.  Also, some C compilers are pedantic and require C-style comments `/* */` rather than allowing the single-line C++ comments, so either remove the comment or change to `/* */`